### DISCUSSION TITLE
Why do Flux models locate in StabilityMatrix-win-x64\Data\Models\unet folder?

### DISCUSSION THREAD

**Discussion by nothingness6** [LykosAI/StabilityMatrix#1060] [Q&A]
Since I downloaded the Flux models, they automatically locate in StabilityMatrix-win-x64\Data\Models\unet folder. However, you will find the Fulx models are missing when you launch ComfyUI. It works fine if you locate them in StabilityMatrix-win-x64\Data\Models\StableDiffusion folder. 
Then, why did you make it like this? Is there any specific reason?

**Comment by mohnjiles** [comment]
That was chosen because the base/original Flux model, as well as the gguf versions and some others, do not come "all-in-one" with the text encoders and VAE baked in, and uses the "Load Diffusion Model" node in ComfyUI (along with some others for loading text encoders & vae).

You may have downloaded an "all-in-one" version with the text encoders & VAE pre-baked into the model, which would need to go in the regular StableDiffusion folder to be loaded by the regular checkpoint loader.

Since we can't really tell from the Civitai metadata, we'll likely add the `StableDiffusion` folder as an option in the "Select Download Location" dropdown. Sorry about that!
